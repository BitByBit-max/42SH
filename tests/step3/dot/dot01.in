echo "echo hello world, this file is executed by dot" > testfile

. ./testfile

rm testfile
