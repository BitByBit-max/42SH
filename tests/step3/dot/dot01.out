hello world, this file is executed by dot
